pkg/logs/wait_for_image.go

```diff
@@ -138,8 +142,12 @@ func (w *imageWaiter) waitBuild(ctx context.Context, writer io.Writer, namespace
 		return "", errors.New("unexpected object received")
 	}
 
-	if build.Status.GetCondition(corev1alpha1.ConditionSucceeded).IsFalse() {
-		return "", errors.Errorf("update to image failed")
+	if condition:= build.Status.GetCondition(corev1alpha1.ConditionSucceeded); condition.IsFalse() {
+		if condition.Message != "" {
```

Comment:
wonder if a single `return "", errors.Wrap(errors.New(condition.Message), "update to image failed")` without the `condition.Message != ""`  check is sufficient? not sure.

Reply:
it will try to wrap still so you get "update to image failed: " with the dangling colon

Reply:
got it. thanks for clarifying